lib/settings.coffee

```diff
@@ -1,8 +1,15 @@
 module.exports =
   config:
+
+    fontSize:
+      title: 'Font Size'
+      description: 'Change the UI font size. Needs to be between 10 and 20.'
+      type: 'string'
```

Comment:
@lee-dohm It seems that by removing the `default`, there is no error but also the setting doesn't  show up.

@benogle `type: ['integer', 'null']` and `default: null` doesn't seem to work, but I tried `type: ['integer', 'string']`. Then the min/max still works and the default can be `Auto` which looks good as a placeholder.

![screen shot 2015-05-28 at 12 30 08 pm](https://cloud.githubusercontent.com/assets/378023/7852278/2fb370ee-0536-11e5-8939-59ed0ae29089.png)